### PR TITLE
Feature/all carbon widget

### PR DESCRIPTION
**Pull request by edbrett**
## Overview

The final climate widget for all carbon (soil and biomass). 

Screenshot 2019-04-05 at 17 57 48

## Demo

Go to dashboards and checkout and country or admin 1/2

## Testing

- Go to climate tab check it loads
- Change the settings menu
- Change location up and down

### CODE REVIEW

File: app/javascript/components/widgets/widgets/climate/carbon-stock/config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  title: 'Carbon stock in {location}',
+  categories: ['climate'],
+  types: ['country'],
+  admins: ['adm0', 'adm1', 'adm2'],
+  options: {
+    variables: ['totalbiomass', 'biomassdensity'],
+    thresholds: true
+  },
+  colors: 'climate',
+  metaKey: '',
```
Comment (blayhem) [inline app/javascript/components/widgets/widgets/climate/carbon-stock/config.js]: It's missing the metadata info

File: app/javascript/data/colors.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   },
   "climate": {
     "ramp": [ "#EDD093", "#BFBC35", "#007F53", "#123A33"],
-    "density": "#895122",
+    "carbon": ["#895122", "#898522", "#892227"],
```
Comment (blayhem) [inline app/javascript/data/colors.json]: These colours (specially #895122 and #892227) look a bit too similar with low contrast or night shift (1.4 contrast ratio)

File: app/javascript/components/widgets/widgets/climate/soil-organic/selectors.js [hunk 3]
```diff
@@ -119,9 +119,9 @@ export const parseSentence = createSelector(
         }
       };
     }
-    const iso = Object.keys(locationsDict).find(
-      key => locationsDict[key] === location
-    );
+    const iso =
+      locationsDict &&
```
Comment (blayhem) [inline app/javascript/components/widgets/widgets/climate/soil-organic/selectors.js]: good catch! I missed a couple of these

File: app/javascript/components/widgets/widgets/climate/carbon-stock/config.js (hunk 1 above)
Comment (edbrett) [reply app/javascript/components/widgets/widgets/climate/carbon-stock/config.js]: We are waiting for them to create it.

File: app/javascript/data/colors.json (hunk 2 above)
Comment (edbrett) [reply app/javascript/data/colors.json]: We chose these from the initial hex code for carbon based on the chroma scale. Waiting for their feedback on these too.

File: app/javascript/components/widgets/widgets/climate/soil-organic/selectors.js (hunk 3 above)
Comment (edbrett) [reply app/javascript/components/widgets/widgets/climate/soil-organic/selectors.js]: They were breaking hot reload so needed fixing.

**Pull request merged**